tree/ntuple/v7/inc/ROOT/libdaos_mock/daos.h

```diff
@@ -257,15 +327,13 @@ typedef struct {
 	char		unused; // silence [-Wextern-c-compat]
 } daos_cont_info_t;
 
-d_rank_list_t *daos_rank_list_parse(const char *str, const char *sep);
-
-int daos_cont_create(daos_handle_t poh, const uuid_t uuid, daos_prop_t *cont_prop,
-		 daos_event_t *ev);
-int daos_cont_open(daos_handle_t poh, const uuid_t uuid, unsigned int flags,
-	       daos_handle_t *coh, daos_cont_info_t *info, daos_event_t *ev);
+int daos_cont_create(daos_handle_t poh, uuid_t uuid, daos_prop_t *cont_prop, daos_event_t *ev);
```

Comment:
We might even remove this decl (unused now); however, if we want to keep it, see the comment in `libdaos_mock.cxx`

Reply:
It would make sense to keep this declaration for development purposes if we do support UUID URIs in `RDaos` as well as label ones, even if the mock library and test suites don't use it.